plugins/pulp_docker/plugins/distributors/distributor_export.py

```diff
@@ -92,9 +92,12 @@ def validate_config(self, repo, config, config_conduit):
         :return: tuple of (bool, str) to describe the result
         :rtype:  tuple
         """
+        repo = model.Repository.objects.get(repo_id=repo_transfer.id)
+        repo.repo_id = repo_transfer.id
```

Comment:
@barnabycourt, do you have a plan in mind to track the use of temp things like this? It would be nice to be able to clean them up right after I get my code in.